packages/client/hmi-client/src/components/workflow/tera-progress-bar.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+	<div>
+		<i class="pi pi-spin pi-spinner"></i>
+		<span>{{ props.status }}...{{ props.value ? '(' + props.value + '%)' : null }}</span>
+		<ProgressBar v-if="props.value" class="progress-bar" :value="props.value"></ProgressBar>
+	</div>
```

Comment:
```suggestion
		<i class="pi pi-spin pi-spinner" />
		<p>{{ status }}...<template v-if="value">{{ `(${value}%)` }}</template></p>
		<ProgressBar v-if="value" class="progress-bar" :value="value" />
```

Reply:
To ben honest you should create a computed for that Status message, instead of doing complex HTML stuff.
```
<p>{{ message }}</p>
...
const message = computed(() => `${props.status}...${ props.value ? `(${value}%)`}` );
```
or something like that.

Reply:
good idea - ill leverage computed here

Reply:
done